web/controllers/event_controller.ex

```diff
@@ -1,30 +1,11 @@
 defmodule Dwylbot.EventController do
   use Dwylbot.Web, :controller
-  alias Dwylbot.Rules, as: DR
-  @github_api Application.get_env(:dwylbot, :github_api)
-  @duration Application.get_env(:dwylbot, :duration)
+  alias Dwylbot.WaitProcess, as: DW
```

Comment:
@SimonLab aliasing is good. prefer a more meaningful keywork though for debugging purposes ... prob not an issue _here_ because there isn't much to go "wrong" in this code, but in other places useful. e.g: `DW` >> `WAIT`  👍